import OutOfRangeError from '../OutOfRangeError';
import createErrorMock from './__ignore__/create-error-mock';

createErrorMock(OutOfRangeError);
